Dockerfile

```diff
@@ -1,13 +1,28 @@
-FROM python:3.9
+FROM python:3.9.2-slim-buster
```

Comment:
This seems to be working for me without problem, and should give a smaller image. But I feel like this is a bit out of scope and I can take this out.